scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/nn/VflNNAggregator.scala

```diff
@@ -82,18 +83,18 @@ class VflNNAggregator(model: Module[Float],
         })
         setReturnMessage(result.toString)
```

Comment:
It does not make sense to return a string; return a lit of tuple (`ValidationMethod.toString`, `ValidationResult.result()._1`)

Reply:
Yes, this is just a tmp logic for now. Will refine afterward referring to orca evaluate and other frameworks like tf/pytorch